src/stream/src/executor_v2/lookup.rs

```diff
@@ -117,11 +125,22 @@ impl ExecutorBuilder for LookupExecutorBuilder {
             .map(|x| OrderPair::new(*x as usize, OrderType::Ascending))
             .collect();
 
+        assert_ne!(
+            node.arrange_fragment_id,
+            u32::MAX,
+            "arrange fragment id is not available"
+        );
+        let arrangement_keyspace_id =
+            unique_operator_id(node.arrange_fragment_id, node.arrange_operator_id);
+
```

Comment:
Perhaps make the calculation of arrangement keyspace a function of `ArrangeExecutor`? Can do it in later PRs.

Reply:
We will move toward using `table_id` in the future. Also we don't have `ArrangeExecutor`. We are simply using `MaterializeExecutor` for storing data.